docs/how-to/contribute.rst

```diff
@@ -453,11 +453,10 @@ code as you go. You could write a comprehensive set of tests for a concrete
 life jacket... but a concrete life jacket would still be useless for the
 purpose it was intended!
 
-As you develop tests and improve coverage, you should be checking that the
-core module is internally **consistent** as well. If you notice any method
+As you develop tests and improve coverage, you should be checking that the core module is internal **consistent** as well. If you notice any method
 names that aren’t internally consistent (e.g., something called ``on_select``
 in one module, but called ``on_selected`` in another), or where the data isn’t
-being handled consistently (one widget updates then refreshes, but another
+being handled consistently (one widget updates then refresh, but another
```

Comment:
This change also doesn't read well. I would remove this change myself, personally.